ucm2/sof-soundwire/sof-soundwire.conf

```diff
@@ -176,3 +176,21 @@ If.HdmiIec61937 {
 	}
 	True.Macro.save_hdmi_cfg.HdmiPCMSave { Name "42-sof-hdmi" }
 }
+
+# Define macros to set up processing controls
+Include.dsp-variables.File "../Intel/sof-hda-dsp/dsp.conf"
+
+# Set BootSequence and FixedBootSequence for drc, eqiir, eqfir
+Macro.SpeakerDrc.SofDrcBootSetup "endpoint='PostMixerSpeaker' direction='Playback'"
+Macro.SpeakerIir.SofEqBootSetup "endpoint='PostMixerSpeaker' direction='Playback' eqtype='Iir'"
+Macro.SpeakerFir.SofEqBootSetup "endpoint='PostMixerSpeaker' direction='Playback' eqtype='Fir'"
+
+# Set BootSequence and FixedBootSequence for Mic array eqiir, tdfb, drc
+Macro.MicArrayIir.SofEqBootSetup "endpoint='MicArray' direction='Capture' eqtype='Iir'"
+Macro.MicArrayDrc.SofDrcBootSetup "endpoint='MicArray' direction='Capture'"
+Macro.MicarrayTdfb.SofBeamformerBootSetup "endpoint='MicArray' direction='Capture'"
+
+# For SoundWire PC with PCH-DMIC set BootSequence and FixedBootSequence for eqiir, tdfb, drc
+Macro.Dmic0ArrayIir.SofEqBootSetup "endpoint='Dmic0' direction='Capture' eqtype='Iir'"
+Macro.Dmic0ArrayDrc.SofDrcBootSetup "endpoint='Dmic0' direction='Capture'"
+Macro.Dmic0ArrayTdfb.SofBeamformerBootSetup "endpoint=Dmic0 direction=Capture"
```

Comment:
Nitpicking: Use 'STR' everywhere or remove '' everywhere.

Reply:
Thanks, this is fixed too.